.github/actions/tpv2-integration-tests/tpv2-integration-tests.ts

```diff
@@ -0,0 +1,31 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+"use strict";
+import child_process, { SpawnSyncOptions, SpawnSyncReturns } from "child_process";
```

Comment:
This is literally the same file that all integration tests use, honestly not even sure why we're using JS runners seems like it's way more code and we do nothing with it.

Reply:
Yeah, you know, I couldn't figure that out either, and at this point I'm just scared to ask since I already approved several personally (I think). I guess I just figured it's what everyone wants to do. I wouldn't bother.